gcip/addons/container/jobs/crane.py

```diff
@@ -5,11 +5,12 @@
 from gcip.core.variables import PredefinedVariables
 from gcip.addons.container.config import DockerClientConfig
 from gcip.addons.container.images import PredefinedImages
+from gcip.addons.container.registries import Registry
 
 
 def copy(
-    src: str,
-    dst: str,
+    src_registry: Union[Registry, str],
+    dst_registry: Union[Registry, str],
```

Comment:
```suggestion
    target_registry: Union[Registry, str],
```

`target` is

- a nice word :)
- short enough to need not to be abbreviated
- crane slang https://michaelsauter.github.io/crane/docs.html